Remove the invoke-extractor lambda

marchuffnagle opened the pull request:
The bitstreams downloader now invokes the textractor lambda directly, then writes to the indexer's SQS queue.

Commit:
Remove the invoke-extractor lambda

paulpilone reviewed (changes requested):
@marchuffnagle This is looking really good and should eliminate a number of potentially duplicate S3 notifications. Can we fix the linter errors and test failures before we merge it in though?

Commit:
Bump max-len up to 90

You're welcome Paul.

Commit:
Update bitstreams-downloader tests

This required creating a LambdaClient interface that was needed so we could pass a fake LambdaClient into the bitstreams downloader tests.

Commit:
DRY-up lambda client creation

marchuffnagle commented on .eslintrc.js:
```diff
@@ -35,7 +35,7 @@ const config = {
       'error',
       {
-        code: 80,
+        code: 90,
```
You're welcome @paulpilone

Commit:
PR cleanup

Commit:
Add tests to ensure that the bitstreams downloader is invoking the textractor

Commit:
Use babel instead of ts-jest

Cuts test time from 18 seconds to 10.

ghost commented:
> @marchuffnagle This is looking really good and should eliminate a number of potentially duplicate S3 notifications. Can we fix the linter errors and test failures before we merge it in though?

Done and ready for review.

Commit:
Fix jest config
